game-app/game-core/src/main/java/games/strategy/engine/framework/startup/ui/PlayerTypes.java

```diff
@@ -127,7 +83,7 @@ public abstract static class Type {
     @Getter(AccessLevel.PACKAGE)
     private final boolean visible;
 
-    Type(final String label) {
+    public Type(final String label) {
```

Comment:
AiType?

Reply:
Not necessarily. This class is used for "human players" and AIs alike.